app/assets/stylesheets/ama_layout/layout_components/breadcrumbs.scss

```diff
@@ -5,4 +5,16 @@
     display: inline-block;
     font-size: 0.75rem;
   }
+
+  a.disabled{
```

Comment:
we're trying to get away from using tag selectors, so this could maybe be a class breadcrumbs__link--disabled instead.